are_we_proc_macro

```diff
@@ -0,0 +1,2 @@
+Hell Yeah - TrueDoctor, 2021
+Hell Yeah - T0mstone, 2021
```

Comment:
This is not a serious addition, it expresses how much we rejoiced when our respective proc macros worked

Reply:
😄